src/Engine/Audio/SteamAudioInterface.cpp

```diff
@@ -37,15 +51,15 @@ SteamAudioInterface::SaveData SteamAudioInterface::SaveFinalizedScene() {
     return saveData;
 }
 
-void SteamAudioInterface::LoadFinalizedScene(SaveData data) {
+void SteamAudioInterface::LoadFinalizedScene(const SaveData& data) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'LoadFinalizedScene' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)